examples/moses-pln-synergy/scm/README.md

```diff
@@ -164,6 +179,10 @@ $5 = (SetLink
    )
 )
 
+### (5) Build the tautology that if X takes treatment-1, then
+    treatment-1 contains compound-A. Semi-formally
+    (\x take(x, treatment-1)) -> (\x contain(treatment-1, compound-A))
```

Comment:
People are used to reading C code. But PLN and atoms are all new, and the exact meaning is not new if one is a beginner. Its hard to tell if the variable name choice is significant or not (ditto, if you did not know C).